bullet/src/EntityManagementFeatures.cc

```diff
@@ -33,31 +33,28 @@ Identity EntityManagementFeatures::ConstructEmptyWorld(
 bool EntityManagementFeatures::RemoveModel(const Identity &_modelID)
 {
   // Check if the model exists
-  if (this->models.find(_modelID) == this->models.end()){
+  if (this->models.find(_modelID.id) == this->models.end()){
```

Comment:
Just leave as it was
```suggestion
  if (this->models.find(_modelID) == this->models.end()){
```

Reply:
Cant, Identity has no operator ==